src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -237,75 +277,27 @@ const AppWrapper = (props: { children: ReactNode }) => {
 
     try {
       await Agent.agent.start();
+      setIsOffline(false);
     } catch (e) {
-      // @TODO - foconnor: Should specifically catch the error instead of all, but OK for now.
-      setAgentInitErr(true);
-      // eslint-disable-next-line no-console
-      console.error(e);
-      return;
+      const errorStack = (e as Error).stack as string;
+      // If the error is failed to fetch with signify, we retry until the connection is secured
+      if (/SignifyClient/gi.test(errorStack)) {
+        Agent.agent.bootAndConnect().then(() => {
+          setIsOffline(!Agent.agent.isAgentReady());
+        });
+      } else {
+        throw e;
+      }
     }
     dispatch(setPauseQueueIncomingRequest(true));
```

Comment:
Are there any things we can do while offline for queue items? I don't think so right?